scripts/docker_files/docker_file_ci_ubuntu_20_04/DockerFile

```diff
@@ -47,12 +47,21 @@ RUN apt-get update -y && apt-get upgrade -y && \
     pip3 install \
         numpy \
         scipy && \
+    # install MMG 5.4
```

Comment:
Shouldn't we remove this version if we have 5.5?

Reply:
in the future after a transition period, so far still 5.4 is the default
otherwise we break everyones code that uses 5.4